packages/rancher-wins-upgrader/charts/values.yaml

```diff
@@ -0,0 +1,40 @@
+# Default values for rancher-windows-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Configuration
+
+global:
+  cattle:
+    systemDefaultRegistry: ""
+
+prefixPath: c:\\
+
+## Contains one or more configurations for the wins server to be applied across all of the nodes based on the nodeSelector and tolerations provided
+##
+winsConfigs:
```

Comment:
takes in multiple wins configs, so you can limit the wins server whitelist and versions across sets of nodes.

This way you aren't forced to have one universal wins config or wins server version across all nodes, if you so wish to do so, and can configure everything based on nodeSelectors and tolerations.

Reply:
This requires one at minimum, right?

Reply:
yup, will add the fix. Good call.